docs-go/Makefile

```diff
@@ -0,0 +1,10 @@
+WEAVE=$(HOME)/go/bin/weave 
+
+all: $(WEAVE) flows.md models.md
```

Comment:
Shouldn't there be a change to a `go:generate` line somewhere?

Reply:
Are you suggesting that a go:generate should call this makefile?
I was thinking that this is something that the docs person runs manually.
But I guess it could be both; we could add a go:generate to the directory where the snippets live.

Done.